src/releng_docs/projects/shipit-code-coverage-crawler.rst

```diff
@@ -0,0 +1,16 @@
+.. shipit-code-coverage-crawler-project:
+
+Project: shipit-code-coverage-crawler
+=====================================
+
+:contact: `Marco Castelluccio`_, (backup `Release Management`_)
+
+This project uses the `Code Coverage Crawler`_ to navigate pages from a list of
+websites and publishes reports of code that is being run by navigating those
```

Comment:
Probably, it is better to say "being run by interacting with elements on those pages".